workspaces/liferay-sample-workspace/gradle.properties

```diff
@@ -9,4 +9,5 @@ liferay.workspace.node.package.manager=yarn
 liferay.workspace.product=dxp-2023.q4.5
 liferay.workspace.themes.dir=themes
 liferay.workspace.wars.dir=modules
+liferay.workspace.home.dir=/home/me/dev/bundles/master
```

Comment:
you can delete this line and create a `gradle-local.properties` which will do the same but be ignored by git

Reply:
Oh didn't know that. I wouldn't send it for real though.

Reply:
Oh I know... I was just letting you in on this little secret workflow improvement :)

Reply:
Now I am part of a secret society :wink: